### ISSUE TITLE
TypeError: arn.startsWith is not a function thrown on webhook construct

### ISSUE THREAD

**Issue by tdjones** [getlift/lift#157]
### Description

I've added a simple webhook construct to my serverless following the examples.

```
constructs:
  mywebhook:
    type: webhook
    authorizer:
      handler: webhook.authorizer
      environment:
        WEBHOOK_SECRET: fefefef
    path: /webhook/wildapricot
```

when I attempt to deploy:
```
TypeError: arn.startsWith is not a function
      at parseArnShape (/projectpath/node_modules/@aws-cdk/core/lib/arn.js:305:30)
      at Function.split (/projectpath/node_modules/@aws-cdk/core/lib/arn.js:130:28)
      at Stack.splitArn (/projectpath/node_modules/@aws-cdk/core/lib/stack.js:388:26)
      at new Resource (/projectpath/node_modules/@aws-cdk/core/lib/resource.js:36:24)
      at new FunctionBase (/projectpath/node_modules/@aws-cdk/aws-lambda/lib/function-base.js:18:9)
      at new Import (/projectpath/node_modules/@aws-cdk/aws-lambda/lib/function.js:316:17)
      at Function.fromFunctionAttributes (/projectpath/node_modules/@aws-cdk/aws-lambda/lib/function.js:338:16)
      at Function.fromFunctionArn (/projectpath/node_modules/@aws-cdk/aws-lambda/lib/function.js:298:25)
      at new Webhook (/projectpath/node_modules/serverless-lift/dist/src/constructs/aws/Webhook.js:112:49)
      at Function.create (/projectpath/node_modules/serverless-lift/dist/src/constructs/abstracts/AwsConstruct.js:30:12)
      at _AwsProvider.createConstruct (/projectpath/node_modules/serverless-lift/dist/src/providers/AwsProvider.js:73:22)
      at _LiftPlugin.loadConstructs (/projectpath/node_modules/serverless-lift/dist/src/plugin.js:181:64)
      at initialize (/projectpath/node_modules/serverless-lift/dist/src/plugin.js:92:14)
      at PluginManager.run (/projectpath/node_modules/serverless/lib/classes/PluginManager.js:610:13)
      at async Serverless.run (/projectpath/node_modules/serverless/lib/Serverless.js:327:5)
      at async /projectpath/node_modules/serverless/scripts/serverless.js:697:9
```

Print statement in fromFunctionArn:
`fromFunctionArn: id: LambdaAuthorizer, functionArn: ${Token[TOKEN.207]}`
I'm not familiar with cdk well enough to know if ${Token[TOKEN.207]} is the correct string to be passed but it looks like arn.js is looking for a string that begins with "arn:"
`const looksLikeArn = arn.startsWith('arn:');`

### How to Reproduce

Add to serverless.yml:

```
constructs:
  mywebhook:
    type: webhook
    authorizer:
      handler: webhook.authorizer
    path: /webhook/target
```

sls deploy

### Additional Information

Using:
Serverless:
Framework Core: 2.42.0 (local)
Plugin: 5.1.2
SDK: 4.2.2
Components: 3.10.0

**Comment by Faslo** [comment]
Having the same issue

**Comment by t-richard** [comment]
Hi!

I think this bug was fixed in the latest version (1.12.0) with this commit https://github.com/getlift/lift/commit/a71a1c49bdf15f42bdf6e51260fe5408b5681319

Could you update and see if it fixes it on your end?

**Comment by mnapoli** [comment]
Oh yes sorry I forgot to update here! Please let me know if the latest version still causes issues.
